src/PoshArmDeployment/Public/ResourceProviders/Microsoft.Logic/New-ArmLogicApp.ps1

```diff
@@ -0,0 +1,40 @@
+function New-ArmLogicApp {
+    [CmdletBinding(SupportsShouldProcess = $true)]
+    [OutputType("LogicApp")]
+    Param(
+        [Parameter(Mandatory, ValueFromPipeline)]
+        [ValidatePattern('^(\[.*\]|[a-zA-Z0-9-]*)$')]
+        [string]
+        $Name,
+        [PSTypeName("WorkflowDefinition")]
+        $WorkflowDefinition = @{},
+        [string]
+        $ApiVersion = '2017-07-01',
+        [string]
+        $Location = $script:Location,
+        [string]
+        $State = 'Enabled'
```

Comment:
Change to a switch named "Disable" and map it to the correct string in this cmdlet

Reply:
I think there is more than 2 states ("NotSpecified", "Completed", "Enabled", "Disabled", "Deleted", "Suspended")
Do we only add support for Enabled and Disabled?

Reply:
haaaa perfect then